discover/portal/articles/210-setting-up/01-system-settings/01-system-settings.markdown

```diff
@@ -0,0 +1,121 @@
+Settings configuration does nothing to this Widget Instance anymore.
+
+If you make some regrettable configuration decisions and can't recall exactly
+what you did, start over by clicking the actions button
+(![Actions](../../../images/icon-actions.png)), then clicking *Reset Default
+Values*. 
+
+## Configuration Scope [](id=configuration-scope)
+
+While browsing the categories of System Settings, you'll notice that clicking
+into a topic (for example, Blogs) reveals entries at different scopes.each entry
+has a *Name* and a *Scope*. This scope is different than the overall
+application configuration scope discussed in the introduction to this section.
+This is the System Settings application after all, so all of the configuration
+done here affects the System scope. The Scope field here refers to the scope at 
+which this default, system-wide configuration is overridden. There are four 
+values that you'll see under Scope: 
+
+- *System:* Any configuration made at the system scope here becomes the final
+  value for the application in a system-wide fashion. It affects the whole
+  system and isn't overridden anywhere else. 
+
+    ![Figure 4: Some System Settings entries have a System scope.](../../../images/system-settings-system-scope.png)
+
+- *Virtual Instance Scope:* Making configuration changes at this level affects
+    the entire Virtual Instance. 
+
+    ![Figure 5: Some System Settings entries have a Default Configuration for Application scope.](../../../images/system-settings-application-scope.png)
+
+- *Site Scope* Configuration at this scope is overridden in each site. 
+
+    ![Figure 6: Some System Settings entries have a Default Configuration for All Sites scope.](../../../images/system-settings-site-scope.png)
+
+- *Widget Scope:* Configuration at this scope is overridden in each
+    Widget Instance (like the Blogs example above). 
+
+    ![Figure 7: Some System Settings entries have a Default Configuration for All Instances scope.](../../../images/system-settings-instance-scope.png)
+
+If an application is configurable from Instance Settings and System Settings,
+use System Settings to configure it whenever possible. If you use Instance
+Settings and later want to revert to using the System Settings default
+configurations, look for the *Reset Values* button from the Instance Settings
+entry. 
+
+![Figure 8: Some Instance Settings entries have a *Reset Values* button so you can safely revert your configuration changes, falling back to the System Settings defaults.](../../../images/instance-settings-reset-values.png)
+
+The Reset Values button removes configuration values from the database
+so you can rely on the defaults specified in System Settings. If there is no
+Reset Values button for an Instance Settings entry, once you use Instance
+Settings to configure the application, you can't later decide to use System
+Settings. Only changes to Instance Settings will be recognized. 
```

Comment:
I don't understand this paragraph.

Reply:
I'm trying to describe the reason for the Reset Values button here.

An example might be best: 
If memory serves, you can configure the same LDAP settings in System Settings and Instance Settings. If you start with Instance Settings, you can never go back to the System Settings. Unless, of course, you use the Reset Values button. 

I'll try to make this paragraph more clear.

Reply:
Is something like this better?

> The Reset Values button removes configuration values from the database so you
> can rely on the defaults that were provided out-of-the-box. If there is no Reset
> Values button for an Instance Settings entry, using Instance Settings to
> configure the application breaks its connection with System Settings. Only
> changes to Instance Settings will be recognized.

Reply:
Hey Russ,

That's not the function of "Reset Values"

What it does is to change the values within System Settings to the defaults that came with the product. If we are talking about a System scope configuration that action effectively resets the configuration. For any other scope, it just resets the defaults for those cases in which there are no specific values for a higher scope.

Makes sense?

PS: If we later add an action to delete the specific configurations, it would not be a button for the form, but an action outside. And we would probably need to provide more information to the user about which specific instances will be affected.